tests/e2e/tools/lib/covers.js

```diff
@@ -28,20 +29,37 @@ async function addCoversBlock(page, style) {
     // Fill in the tags.
     await page.locator('input[placeholder="Select Tags"]').type(TAG_NAMES[0]);
     await page.locator('li.components-form-token-field__suggestion').click();
-    await page.locator('input.components-form-token-field__input').type(TAG_NAMES[1]);
+    await page.locator('input.components-form-token-field__input').nth(0).type(TAG_NAMES[1]);
     await page.locator('li.components-form-token-field__suggestion').click();
-    await page.locator('input.components-form-token-field__input').type(TAG_NAMES[2]);
+    await page.locator('input.components-form-token-field__input').nth(0).type(TAG_NAMES[2]);
     await page.locator('li.components-form-token-field__suggestion').click();
-    await page.locator('input.components-form-token-field__input').type(TAG_NAMES[3]);
+    await page.locator('input.components-form-token-field__input').nth(0).type(TAG_NAMES[3]);
     await page.locator('li.components-form-token-field__suggestion').click();
   }
 
+  // Default style, i.e 'Content cover'.
+  if (style === '') {
```

Comment:
Maybe instead of using an empty string and having to add this comment to clarify, we could set it as default to `Content`?

Reply:
Or we could just pass down `Content` style parameter in the calls

Reply:
'Content' word not match in covers style, hence pass a 'Default'.
![image](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/assets/5357471/4d875412-2f70-4ac7-88c3-fc1d78fbadfe)

Reply:
Ah, I should have known that, sorry 😬